src/bindings/python/BUILDING.md

```diff
@@ -1,88 +1,145 @@
+```bash
+apt install git wget build-essential cmake
+apt install python3 python3-dev python3-pip python3-virtualenv python-is-python3
+```
 
 On macOS, you can use [Homebrew](https://brew.sh) to install required packages:
 
-    brew install cmake
-    brew install automake
-    brew install libtool
-    brew install python3
+```bash
+brew install cmake
+brew install automake
+brew install libtool
+brew install python3
+```
 
 Install Cython in the Python installation, or virtualenv that you are planning to use:
 
-    pip3 install cython
+```bash
+pip3 install cython
+```
 
- ### Configure and Build as a part of OpenVINO™ Toolkit on Linux and macOS
+### Configure and Build as a part of OpenVINO™ Toolkit on Linux and macOS
 
 The following section illustrates how to build and install OpenVINO™ in a workspace directory using CMake.
-The workspace directory is specified by the `${OPENVINO_BASEDIR}` variable. Set this variable to a directory of your choice:
+The workspace directory is specified by the `${OV_WORKSPACE}` variable. Set this variable to a directory of your choice:
 
-    export OPENVINO_BASEDIR=/path/to/my/workspace
+```bash
+export OV_WORKSPACE=/path/to/my/workspace
+```
 
 Now you can clone the OpenVINO™ repository, configure it using `cmake` and build using `make`. Please note that we're disabling
-the building of a few modules by setting the `ENABLE_*` flag to `OFF`. In order to build the OpenVINO™ Python APIs
-set the mentioned flags to `ON`. Note the `CMAKE_INSTALL_PREFIX`, which defaults to `/usr/local/` if not set.
-
-    cd "${OPENVINO_BASEDIR}"
-    git clone --recursive https://github.com/openvinotoolkit/openvino.git
-    mkdir openvino/build
-    cd openvino/build
-
-    cmake .. \
-        -DENABLE_INTEL_GPU=OFF \
-        -DENABLE_OPENCV=OFF \
-        -DENABLE_INTEL_MYRIAD_COMMON=OFF \
-        -DENABLE_PYTHON=ON \
-        -DENABLE_OV_ONNX_FRONTEND=ON \
-        -DCMAKE_INSTALL_PREFIX="${OPENVINO_BASEDIR}/openvino_dist"
-
-    make -j 4
-    make install
-
-The Python module is installed in the `${OPENVINO_BASEDIR}/openvino_dist/python/python<version>/` folder.
+the building of a few modules and plugins by setting the `ENABLE_*` flag to `OFF`.
+
+In order to build the OpenVINO™ Python APIs set the `ENABLE_PYTHON` flag to `ON`.
+
+Note the `CMAKE_INSTALL_PREFIX`, which defaults to `/usr/local/` if not set.
+
+```bash
+cd ${OV_WORKSPACE}
+git clone --recursive https://github.com/openvinotoolkit/openvino.git
+cd openvino
+git submodule update --init
+
+### You may also want to install OpenVINO™ dependencies with:
+# ./install_build_dependencies.sh
+
+mkdir build && cd build
+
+cmake .. \
+-DCMAKE_BUILD_TYPE=Release \
+-DENABLE_INTEL_CPU=ON \
+-DENABLE_INTEL_GNA=OFF \
+-DENABLE_INTEL_GPU=OFF \
+-DENABLE_INTEL_MYRIAD_COMMON=OFF \
+-DENABLE_INTEL_MYRIAD=OFF \
+-DENABLE_HETERO=OFF \
+-DENABLE_MULTI=OFF \
+-DENABLE_AUTO=ON \
+-DENABLE_TEMPLATE=ON \
```

Comment:
Why do we need to have all these cmake flags? Can we use default configuration in order to don't scare our users :)

Reply:
It is simply hard to keep all md files for all the components in sync. I vote for at least using default values. At most I would love to see all necessary information in wiki. This file doesn't seem really visible

Reply:
> This file doesn't seem really visible

It is located in top-level bindings folder. Following this specific argument, maybe a root cause is Python folder being hidden itself? We talked about it... should we go over it again? 😄

> Why do we need to have all these cmake flags? Can we use default configuration in order to don't scare our users :)

To make it compact and focus on some flags that are "involved" in Python API as we have onnx-based tests in our repository. There is a comment above, if someone is interested in developing/building -- there is clear comment for it.

Reply:
Discussed and adjusted offline! 👍